### PR TITLE
Bump version and update CHANGELOG for core v2.15.0

### PR DESCRIPTION
**Pull request by chainchad**
Signed-off-by: chainchad <96362174+chainchad@users.noreply.github.com>

### CODE REVIEW

**Review by momentmaker — DISMISSED** [review]

File: .changeset/config.json [hunk 1]
```diff
@@ -1,11 +1,6 @@
 {
   "$schema": "https://unpkg.com/@changesets/config@2.3.1/schema.json",
-  "changelog": [
-    "@changesets/changelog-github",
-    {
-      "repo": "smartcontractkit/chainlink"
```
Comment (momentmaker) [inline .changeset/config.json]: hmm why was this removed?

**Review by momentmaker — APPROVED** [review]

**Review by erikburt — APPROVED** [review]
